build-tools/automation/azure-pipelines.yaml

```diff
@@ -65,6 +65,8 @@ variables:
   DotNetNUnitCategories: '& TestCategory != DotNetIgnore & TestCategory != AOT & TestCategory != MkBundle & TestCategory != MonoSymbolicate & TestCategory != PackagesConfig & TestCategory != StaticProject'
   NUnit.NumberOfTestWorkers: 4
   GitHub.Token: $(github--pat--vs-mobiletools-engineering-service2)
+  # Temporary workaround for: https://github.com/dotnet/msbuild/issues/5863
+  MSBuildDisableFeaturesFromVersion: 16.10
```

Comment:
We probably shouldn't merge this until we get a build with a fix for: https://github.com/dotnet/msbuild/issues/5863

It would require everyone to set this env var when doing development.

Reply:
/cc @rolfbjarne this is that env var if you need it.